frontend/services/entities/invites.ts

```diff
@@ -64,10 +55,10 @@ export default {
 
     let searchQuery = "";
     if (globalFilter !== "") {
-      searchQuery = `&query=${globalFilter}`;
+      searchQuery = `query=${globalFilter}`;
     }
 
-    const path = `${INVITES}?${pagination}${searchQuery}${orderKeyParam}${orderDirection}`;
+    const path = `${INVITES}?${searchQuery}${orderKeyParam}${orderDirection}`;
```

Comment:
Can we take the opportunity to improve this and use `buildQueryStringFromParams` method?

Reply:
Yeah, I think I got time today, going to put another bug PR up and circle back